test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -84,13 +89,29 @@
                     return result;
                 }
             };
+        },
+        toMatchIEEE754Number: function (util, customEqualityTesters) {
+            return {
+                compare: function (actual, expected) {
+                    var result = {};
+                    if (typeof actual === 'number' && typeof expected === 'number') {
+                        result.pass = Object.is(actual, expected);
```

Comment:
These tests are validating exact representations of values can be serialized and deserialized from Vireo memory. We should not be using epsilons for comparisons during these tests.